src/aap_eda/wsapi/consumers.py

```diff
@@ -72,12 +72,33 @@ class Event(Enum):
 DATETIME_FORMAT = "%Y-%m-%dT%H:%M:%S.%f%z"
 
 
+class DefaultConsumer(AsyncWebsocketConsumer):
+    """Default consumer for websocket connections.
+
+    This is the consumer to handle all the unexpected paths, it will close the
+    connection with an error message.
+    """
+
+    async def connect(self):
+        await self.accept()
+        await self.send('{"error": "invalid path"}')
+        await self.close()
+
+
 class AnsibleRulebookConsumer(AsyncWebsocketConsumer):
     async def receive(self, text_data=None, bytes_data=None):
         data = json.loads(text_data)
         logger.debug(f"AnsibleRulebookConsumer received: {data}")
 
-        msg_type = MessageType(data.get("type"))
+        msg_type = data.get("type")
+        try:
+            msg_type = MessageType(msg_type)
+        except ValueError:
+            logger.error(f"Unsupported message type: {data}")
+            payload = {"error": f"unsupported message type: {msg_type}"}
+            await self.send(text_data=json.dumps(payload))
```

Comment:
@Alex-Izquierdo What are we sending back?

Reply:
I think we should just terminate the session and not send anything back

Reply:
updated